### ISSUE TITLE
Get text from select field instead of id (key)

### ISSUE THREAD

**Issue by Pablinus** [reduxframework/redux-framework#2130]
Is it possible to get the text in options instead of the id ("key")? For example.

    $fields = array(
        'id'       => 'opt-select',
        'type'     => 'select',
        'title'    => __('Select Option', 'redux-framework-demo'), 
        'subtitle' => __('No validation can be done on this field type', 'redux-framework-demo'),
        'desc'     => __('This is the description field, again good for additional info.', 'redux-framework-    demo'),
        // Must provide key => value pairs for select options
        'options'  => array(
            '1' => 'Opt 1',
            '2' => 'Opt 2',
            '3' => 'Opt 3'
        ),
        'default'  => '2',
    );
Instead of get

    global $redux_demo;
    echo 'Single Select value: ' . $redux_demo['opt-select']; 
This will return 1, 2, 3. But I want to get Opt 1, Opt 2, Opt 3. Is this possible?

**Comment by kprovance** [comment]
I don't believe so.  The alternative is to write a function to select the name from the index, using it/then

**Comment by kprovance** [comment]
Oh wow, I completely misanswered this.  let me try again:  You can use strings, you'd use them in place of indexes.

```php
$fields = array(
    'id'       => 'opt-select',
    'type'     => 'select',
    'title'    => __('Select Option', 'redux-framework-demo'), 
    'subtitle' => __('No validation can be done on this field type', 'redux-framework-demo'),
    'desc'     => __('This is the description field, again good for additional info.', 'redux-framework-    demo'),
    // Must provide key => value pairs for select options
    'options'  => array(
        'opt-1' => 'Opt 1',
        'opt-2' => 'Opt 2',
        'opt-3' => 'Opt 3'
    ),
    'default'  => 'opt-2',
);
```
Sorry.  Long morning.  ;-)

**Comment by Pablinus** [comment]
Yes thats work Kevin. Thanks a lot!!!

**Comment by dovy** [comment]
If you like Redux, or the support we render, please do one or all of the following:
* Donate to keep us going: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=MMFMHWUPKHKPW
* Review us: http://wordpress.org/support/view/plugin-reviews/redux-framework
* Like us: http://facebook.com/ReduxFramework
* Follow us: http://twitter.com/ReduxFramework
